### DISCUSSION TITLE
Different types of HVDC links?

### DISCUSSION THREAD

**Discussion by victormbarriosb** [e2nIEE/pandapower#1076] [Q&A]
Hi, I was trying to get some results with pandapower but, there is only one DC model, I would like to contribute the other models to the library in case I finish with the algorithm for those models. These would be my first contributions on this platform. Is there a specific rule to follow?

**Comment by lthurner** [comment]
Sorry for the late response - any contributions in this area would be highly appreciated. There is a technical guide on how to contribute here: https://github.com/e2nIEE/pandapower/blob/develop/CONTRIBUTING.rst. If you have any further questions, feel free to ask!
